Wiki/FAQ.md

```diff
@@ -0,0 +1,709 @@
+
+If you have no backup you will have to do the -salvagewallet as starting
+option.
+
+# Sync
+
+## I have lots of connections, but synchronizing stopped at a certain block. What should I do?
+
+Try this:
+
+1.  Make sure Gridcoin is running as an administrator.
+2.  Close the client
+3.  Clean out your Gridcoin Appdata folder EXCEPT wallet.dat, your
+    GridcoinResearch.conf file and the folders walletbackups and testnet
+    if present. For windows '%appdata%/Roaming/GridcoinResearch', For
+    Linux '\~/.GridcoinResearch'
+4.  You can also get the snapshot [Here](http://download.gridcoin.us/download/downloadstake/signed/snapshot.zip)
+    and extract it to the Gridcoin data folder. For windows -\>
+    '%appdata%/Roaming/GridcoinResearch', For Linux
+    '\~/.GridcoinResearch.
+
+There is also an [official
+thread](https://cryptocointalk.com/topic/20303-gridcoin-proof-of-research-connection-sync-problem-thread/)
+for sync problem if it didn't work.
+
+## How could I test whether I am in sync with the network?
+
+Compare your highest block number with the one listed at
+<http://www.gridresearchcorp.com/gridcoin/>
+
+When the explorer is clearly ahead of you (\> 9 blocks) and increasing
+about every 90 seconds, your node is most likely on the fork. You can
+wait till it syncs or rebuild the block chain.
+
+When you are clearly ahead of the explorer and / or the explorer has
+stopped incrementing on a regular basis, it's more likely that the
+explorer will sync up with you and your connected nodes at some point.
+
+## How to get back in sync fast?
+
+Ensure you have upgraded to latest wallet version.
+
+Try using the snapshot you can get it [Here](http://download.gridcoin.us/download/downloadstake/signed/snapshot.zip).
+While it downloads, delete the peers.dat in
+%appdata%\\Roaming\\GridcoinResearch - you may have a bad peer group.
+
+While the client restarts after the download blocks, you can also try to
+switch off/on your router if you get a new IP with that, to make sure
+you are not banned by good nodes because of earlier behavior.
+
+## I certainly fell off the consensus chain, how to get back on?
+
+1. ensure you are running the latest version
+
+2. try to restart the client
+
+3. try to use the [snapshot](http://download.gridcoin.us/download/downloadstake/signed/snapshot.zip).
+
+If you tried each of the steps above and are sure you are running the
+correct version
+
+Backup your wallet.dat file
+
+Close the Gridcoin-Research wallet
+
+In the roaming folder delete all but gridcoinresearch.conf and
+wallet.dat and the wallet backup folder.
+
+Check your [List of Addnodes](List_of_Addnodes "wikilink")
+
+Start Gridcoin-Research and wait. If you continue to have trouble visit
+the technical support forum:
+<https://cryptocointalk.com/forum/2380-technical-support/>
+
+## I still can't get connections. How can I test, whether the net is down or my node is misconfigured?
+
+Open Windows/Linux command box and enter "telnet node.gridcoin.us

```

Comment:
This is no longer online. Should use some other check

Reply:
What other check would you recommend?

Reply:
@nathanielcwm I'd use `telnet addnode-us-central.cycy.me 32749`